### PR TITLE
fix: undercount offline peers

### PR DESCRIPTION
**Pull request by dennis-tra**

### CODE REVIEW

File: config/plotdefs/dht-availability-classified-overall.yaml [hunk 1]
```diff
@@ -3,25 +3,35 @@ datasets:
...
-      select date,
+      select u.date,
         sum(case when uptime >= 0.8 then 1 else 0 end) as online,
         sum(case when uptime >= 0.4 and uptime < 0.8 then 1 else 0 end) as mostlyonline,
-        sum(case when uptime >= 0.1 and uptime < 0.4 then 1 else 0 end) as mostlyoffline,
-        sum(case when uptime < 0.1 then 1 else 0 end) as offline
-      from u
-      group by date
-      order by date;
+        sum(case when uptime < 0.4 then 1 else 0 end) as mostlyoffline,
```
Comment (dennis-tra) [inline config/plotdefs/dht-availability-classified-overall.yaml:30]: note, redefined mostlyoffline. If we're okay with that we need to update the copy as well

Comment (iand) [reply config/plotdefs/dht-availability-classified-overall.yaml:30]: I don't think we should redefine.

Comment (dennis-tra) [reply config/plotdefs/dht-availability-classified-overall.yaml:30]: so then we must add to the count 👍

Comment (iand) [reply config/plotdefs/dht-availability-classified-overall.yaml]: why not union with something like

```
select 0 as online, 0 as mostlyonline, 0 as mostlyoffline, count(distinct o.peer_id) as offline
from select
          period.end as date,
          v.peer_id
        from period, visits v
        where v.visit_started_at <@ period.range
        group by period.end, v.peer_id
        having bool_and(v.connect_error IS not null)
```

II haven't tried so possibly we'd have to repeat the `with period`  part too

**Comment by iand** [comment]
Also need to update dht-availability-classified-current.yaml and dht-availability-classified-region.yaml

**Pull request closed without merging**